components/scream/data/scream_default_output.yaml

```diff
@@ -60,4 +60,7 @@ Output Control:
   Timestamp in Filename: true
   AVG Type in Filename: false
   Frequency in Filename: false
+Checkpoint Control:
+  Frequency: ${REST_N}
```

Comment:
Aren't `REST_XYZ` options for the model restart? Shouldn't we instead use `HIST_OPTION` and `HIST_N` from cime?

Reply:
No I think we still want `REST_N`.  If we use `HIST_N` then we will never need to checkpoint because that would correspond to when we write output.  But if HIST_N != REST_N and we write a restart file we will want a corresponding checkpoint to accompany it, in case we restart the run from that file.